Proofread docs v1.4 (#296)
AY2021S1-CS2103T-T10-4/tp: docs/UserGuide.md

<div markdown="block" class="alert alert-warning">

:warning: Boxes with the :warning: icon contain important information on how to keep your Tutor's Pet data safe.

</div>

--------------------------------------------------------------------------------------------------------------------

## Quick Start
This section provides information on how to quickly start using Tutor's Pet.

### Installing Tutor's Pet

Here are a few steps to get you started on Tutor's Pet:

1. Ensure you have **Java 11** or above installed in your computer.

1. Download the latest version of Tutor's Pet [here](https://github.com/AY2021S1-CS2103T-T10-4/tp/releases).

> 1. Copy the downloaded Tutor's Pet to a folder of your choosing. This shall be known as the home folder of your Tutor's Pet.

Reviewer: ```suggestion
1. Copy the downloaded Tutor's Pet to a folder of your choice. This is known as the home folder of your Tutor's Pet.
```

1. Double-click the file to start the application. An application similar to the one below should appear in a few
   seconds.<br>

   ![Ui](images/Ui.png)

### Using Tutor's Pet

This section offers an overview of the layout in Tutor's Pet.

> There are four main areas in Tutor's Pet:

Reviewer: Is there are particular reason as to why we want to split the result display vs command box?

Author: It looks more intuitive to split the boxes and the screenshot got the different labels also.

<br/>

* The **Help** tab consists of a help button. If you require any assistance, click on the help button, copy the link
 displayed and paste it into any web browser. Alternatively, press <kbd>F1</kbd> to bring up the help window.

--------------------------------------------------------------------------------------------------------------------

## Commands

Commands are the main way you interact with your Tutor's Pet.

This section provides information about the [command format](#command-format), [overview](#command-overview) of available commands,
and detailed instructions on how to use each command.

### Command Format

<div markdown="block" class="alert alert-info">

**:information_source: Notes about the command format:**<br>

This section details the format of the commands available in Tutor's Pet. We will adhere to the following:

* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
  e.g. In `add-student n\NAME`, `NAME` is a parameter which can be used as `add-student n\John Doe`.

* Items in square brackets are optional.<br>
  e.g `n\NAME [tag\TAG]` can be used as `n\John Doe tag\student` or as `n\John Doe`.

> * Items with `…` after them can be used multiple or zero times.<br>

Reviewer: Would this be better?
```suggestion
* Items with `…` after them can be used zero or more times.<br>
```

For example:
1. After finding the class CS2103T and student Alex, you wish to view all your students and classes.

1. You type in the command `list` and press <kbd>Enter</kbd>.

   ![List before](images/ugimages/ListBefore.png)

1. Now, you can see all your students and classes.

   ![List after](images/ugimages/ListAfter.png)

#### Undoing previous changes : `undo`

If you have accidentally entered a command that permanently modifies the data in your Tutor's Pet,

> you can use this command to undo the modification.

Reviewer: I would be inclined to retain the term `change` instead since it has been used in multiple other places like view change history etc.

#### Redoing previously undone changes : `redo`

> If you decide that you do not need to undo a command, you can use `redo` to restore the command.

Reviewer: I'm not sure if it would be good to somewhat distinguish between a change vs a command?

Author: I switched to command because we are undoing and redoing commands. How should I distinguish?

Reviewer: If we want to suggest that we are undoing/redoing commands instead of changes, then we would have to rephrase the view change history as well since that would then imply that you are viewing a command history. But that itself could become confusing with the recall feature.

Other examples:
* `find-class CS2030 lab`<br>
  `edit-class 1 n\CS2030 lab L05`<br>
  Edits the name of the 1st class in the result of the find command.

<div markdown="block" class="alert alert-primary">

> :bulb: **Tip:** Make use of the `find-class` command to filter the displayed class list to your

Reviewer: Do we want to put a hyperlink to the find class command?

**:information_source: Notes about the command:**<br>

  * You must type out the day in full.

>   * You are not allowed to type in overnight lessons. For example, a lesson cannot start at `23:59` and end at `00:01`.

Reviewer: ```suggestion
  * You are not allowed to add overnight lessons. For example, a lesson cannot start at `23:59` and end at `00:01`.
```

* Gives a summary of the student's attendance at the specified `CLASS_INDEX` + `STUDENT_INDEX`.

For example:
1. It is the end of the semester and time for you to grade your students' overall class participation.

> 1. You type in the
>    command `list-student c\1` to see the students in your `CS2103T Tutorial` class.

Reviewer 2: ```suggestion
1. You type in the command `list-student c\1` to see the students in your `CS2103T Tutorial` class.
```
